src/dex/alpaca/alpaca-pool.ts

```diff
@@ -0,0 +1,462 @@
+        AlpacaEventPool.getterFacetInterface.encodeFunctionData('stableTaxBps'),
+    });
+    calls.push({
+      target: params.poolDiamond,
+      callData:
+        AlpacaEventPool.getterFacetInterface.encodeFunctionData('swapFeeBps'),
+    });
+    calls.push({
+      target: params.poolDiamond,
+      callData:
+        AlpacaEventPool.getterFacetInterface.encodeFunctionData('taxBps'),
+    });
+    calls.push({
+      target: params.poolDiamond,
+      callData: AlpacaEventPool.getterFacetInterface.encodeFunctionData(
+        'liquidityOf',
+        [tokenAddress],
+      ),
+    });
+    calls.push({
+      target: params.poolDiamond,
+      callData: AlpacaEventPool.getterFacetInterface.encodeFunctionData(
+        'getStrategyDeltaOf',
+        [tokenAddress],
+      ),
+    });
+    calls.push({
+      target: params.poolDiamond,
+      callData: AlpacaEventPool.getterFacetInterface.encodeFunctionData(
+        'reservedOf',
+        [tokenAddress],
+      ),
+    });
+    calls.push({
+      target: params.poolDiamond,
+      callData: AlpacaEventPool.getterFacetInterface.encodeFunctionData(
+        'guaranteedUsdOf',
+        [tokenAddress],
+      ),
+    });
+    calls.push({
+      target: params.poolDiamond,
+      callData: AlpacaEventPool.getterFacetInterface.encodeFunctionData(
+        'shortSizeOf',
+        [tokenAddress],
+      ),
+    });
+    calls.push({
+      target: params.poolDiamond,
+      callData: AlpacaEventPool.getterFacetInterface.encodeFunctionData(
+        'shortAveragePriceOf',
+        [tokenAddress],
+      ),
+    });
+    calls.push({
+      target: params.poolDiamond,
+      callData: AlpacaEventPool.getterFacetInterface.encodeFunctionData(
+        'tokenMetas',
+        [tokenAddress],
+      ),
+    });
+    calls.push({
+      target: params.poolDiamond,
+      callData:
+        AlpacaEventPool.getterFacetInterface.encodeFunctionData(
+          'isDynamicFeeEnable',
+        ),
+    });
+    calls.push({
+      target: params.poolDiamond,
+      callData:
+        AlpacaEventPool.getterFacetInterface.encodeFunctionData(
+          'additionalAum',
+        ),
+    });
+    calls.push({
+      target: params.poolDiamond,
+      callData:
+        AlpacaEventPool.getterFacetInterface.encodeFunctionData(
+          'discountedAum',
+        ),
+    });
+    calls.push({
+      target: params.poolDiamond,
+      callData: AlpacaEventPool.getterFacetInterface.encodeFunctionData(
+        'getFundingFeeAccounting',
+      ),
+    });
+
+    return calls;
+  }
+
+  private static _genLatestPriceFeedsURL(config: DexParams): string {
+    let url = config.latestPriceFeedsURL + '?';
+    let pythIds = '';
+    for (const token of Object.values(alpacaPoolTokens.poolTokens)) {
+      pythIds += 'ids[]=' + token.PriceId + '&';
+    }
+    url += pythIds + 'verbose=false&binary=true';
+    return url;
+  }
+
+  public static async _getPrices(
+    dexHelper: IDexHelper,
+    config: DexParams,
```

Comment:
then after removing `static` no need to send dexHelper and `config` as a parameter